fdbserver/BlobWorker.actor.cpp

```diff
@@ -385,6 +387,23 @@ struct BlobWorkerData : NonCopyable, ReferenceCounted<BlobWorkerData> {
 		}
 	}
 
+	void addGRVVersion(Version readVersion) {
```

Comment:
nit - maybe we should call it addGRV()? GRV is already an abbreviation of GetReadVersion, so no need to have Version in the function name anymore?

Reply:
Yeah that's a fair point